src/Datadog.Trace.ClrProfiler.Managed/Integrations/ServiceStackRedisIntegration.cs

```diff
@@ -126,5 +119,17 @@ private static string GetRawCommand(byte[][] cmdWithBinaryArgs)
                         }
                     }));
         }
+
+        /*
+         * Ducktyping types
+         */
+#pragma warning disable CS1591 // Missing XML comment for publicly visible type or member
+#pragma warning disable SA1600 // Elements must be documented
```

Comment:
You could document these instead of disabling the warning 😉

Reply:
done in https://github.com/DataDog/dd-trace-dotnet/pull/985/commits/bf1cab2035da1c82eac4b7f68784feb8ad477565 :)